plugins/woocommerce/client/admin/client/task-lists/fills/Marketing/Plugin.tsx

```diff
@@ -26,8 +27,30 @@ export type PluginProps = {
 	manageUrl?: string;
 	name: string;
 	slug: string;
+	tags?: TagsSlug[];
+	learnMoreLink?: string;
+	installExternal?: boolean;
 };
 
+const tagsToPillsMap = {
+	marketplace: __( 'Marketplace', 'woocommerce' ),
+};
+
+function getPills( tags?: TagsSlug[] ) {
```

Comment:
Would it be better to convert this to a React component? or using

```tsx
			{ tags?.map( ( tag ) => 
						tagsToPillsMap[ tag ] && (
							<Pill key={ tag }>{ tagsToPillsMap[ tag ] }</Pill>
						)
					) }
```

Reply:
Sounds good! Addressed in ae32a7c